acc-testcases/README.md

```diff
@@ -0,0 +1,166 @@
+# Running and Writing Acceptance Tests for VMaaS
+- [Running an Acceptance Test](#running-an-acceptance-test)
+- [Writing an Acceptance Test](#writing-an-acceptance-test)
+    - [Writing Acceptance Test for New Resource/Datasource](#writing-acceptance-test-for-new-resource-datasource)
+- [Writing Acceptance Test Suite](#writing-acceptance-test-suite)
+    - [Acceptance Test Suite folder Structure](#acceptance-test-suite-folder-structure)
+
+## Running an Acceptance Test
+
+Acceptance tests can be run using the `acceptance` target in the Terraform
+`Makefile`. Prior to running acceptance test you may need to set provider
+IAM configuration as environment.
+
+```bash
+export IAM_SERVICE_URL=<iam_service_url>
+export TENANT_ID=<tenant_id>
+export USER_SECRET=<user_secret>
+export USER_ID=<user_id>
+```
+
+By Default acceptance test will run test cases from folder `acc-testcases`.
+If you need to specify different test suite or folder, please set environment
+`TF_ACC_TEST_PATH`
+
+For example:
+```bash
+export TF_ACC_TEST_PATH=$PWD/mytestcases
+make acceptance
+```
+above script will run acceptance test from specified folder and it is expected that
+you will cover all the test cases there.
+
+
+## Writing an Acceptance Test
+
+Acceptance test should be written using vmaas acceptance test framework or `atf`. This
+framework supports writing test suites for both resources and data sources.
+
+### Writing Acceptance Test for New Resource/Datasource
+
+Create new test file wil the format `<resource/data_source>_<name of the resource/data source>_test.go`.
```

Comment:
an example of the same format would help.

Reply:
Done